src/main/resources/application-local.yml

```
@@ -0,0 +1,28 @@
+  # JPA
+  jpa:
+    hibernate:
+      ddl-auto: update
+
+# 파일 저장 경로(로컬 서버 내부용)
+item:
+  imgLocation: C:/shop/item
+
+# 개발 환경인 Windows의 일반적인 업로드 경로
+upload:
+  path: file:///C:/shop/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)